windows/wix.json

```
@@ -32,6 +32,19 @@
   "directories": [
     {
       "name": "plugins"
+    },
+    {
+      "name": "storage"
+    }
+  ],
+  "environments": [
+    {
+      "name": "OIQ_OTEL_COLLECTOR_STORAGE",
+      "value": "C:\\Program Files\\observIQ OpenTelemetry Collector",
```

```suggestion
      "value": "[INSTALLDIR]storage",
```
We should do this so that changing the install dir during installation doesn't break anything